src/libs/TransactionUtils.ts

```diff
@@ -266,8 +266,8 @@ function getDescription(transaction: OnyxEntry<Transaction>): string {
 /**
  * Return the amount field from the transaction, return the modifiedAmount if present.
  */
-function getAmount(transaction: OnyxEntry<Transaction>, isFromExpenseReport: boolean): number {
-    // IOU requests cannot have negative values but they can be stored as negative values, let's return absolute value
+function getAmount(transaction: OnyxEntry<Transaction>, isFromExpenseReport = false): number {
```

Comment:
There are a bunch of places where this second parameter is not provided. So I've set it to default `false` to avoid "Wrong params count" errors.